epictrack-api/src/api/reports/anticipated_schedule_report.py

```diff
@@ -132,6 +134,8 @@ def _fetch_data(self, report_date):
                 Work.work_state.in_(
                     [WorkStateEnum.IN_PROGRESS.value, WorkStateEnum.SUSPENDED.value]
                 ),
+                # Filter out specific WorkPhase names
+                ~WorkPhase.name.in_(exclude_phase_names)
```

Comment:
can we try using notin_ or not_in instead of ~ then in_

Reply:
i think this is pretty much a standard to use ~...  
<img width="779" alt="image" src="https://github.com/bcgov/EPIC.track/assets/39633456/d08a6763-85fc-46f5-a79f-f2220c0ed264">